spec/unit/provider/package/cab_spec.rb

```diff
@@ -18,7 +18,7 @@
 
 require "spec_helper"
 
-describe Chef::Provider::Package::Cab do
+describe Chef::Provider::Package::Cab, :windows_only do
```

Comment:
I unit tested that on my local without adding filter `:windows_only` and it ran properly however Travis was failing since it was using linux environment and file-path was mismatched.
See https://travis-ci.org/chef/chef/jobs/192942094#L2093 .